types/feathersjs__feathers/index.d.ts

```diff
@@ -12,7 +12,7 @@
 import { EventEmitter } from 'events';
 import * as self from '@feathersjs/feathers';
 
-declare const feathers: (() => Application<object>) & typeof self;
+declare const feathers: (() => Application) & typeof self;
```

Comment:
Seems like this should be a generic function `<T>() => Application<T>`? In loosely typed JS code that gets you `Application<any>`, in TS code you get type safety.